frontend/src/components/tables/MarketTables.jsx

```diff
@@ -27,104 +24,95 @@ function MarketsTable() {
-            <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-              Trade
-            </th>
-            <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-              🪙
-            </th>
-            <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-              Question
-            </th>
-            <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-              📅 Closes
-            </th>
-            <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-              Creator
-            </th>
-            <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-              👤 Users
-            </th>
-            <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-              📊 Size
-            </th>
-            <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-              💬
-            </th>
-          </tr>
-        </thead>
-        <tbody className='bg-primary-background divide-y divide-gray-200'>
-          {marketsData.map((marketData, index) => (
-            <tr key={index}>
-              <td className='px-6 py-4 text-white'>
-                <Link to={`/markets/${marketData.market.id}`}>⬆️⬇️</Link>
-              </td>
-              <td className='px-6 py-4 text-sm text-gray-500'>
-                {marketData.lastProbability.toFixed(3)}
-              </td>
-              <td className='px-6 py-4 text-sm font-mono text-gray-500'>
-                <Link
-                  to={`/markets/${marketData.market.id}`}
-                  className='text-blue-600 hover:text-blue-800'
-                >
-                  {marketData.market.questionTitle}
-                </Link>
-              </td>
-              <td className='px-6 py-4 text-sm text-gray-500'>
-                {new Date(
-                  marketData.market.resolutionDateTime
-                ).toLocaleDateString()}
-              </td>
-              <td className='px-6 py-4 text-sm text-gray-500'>
-                <Link
-                  to={`/user/${marketData.creator.username}`}
-                  className='text-blue-600 hover:text-blue-800 flex items-center'
+    <div className='w-full max-w-7xl mx-auto  p-1 sm:px-6 lg:px-8'>
+      <h1 className='text-2xl font-semibold text-gray-300 mb-6'>Markets</h1>
+      <div className='bg-gray-800 shadow-md rounded-lg overflow-hidden'>
+        <div className='overflow-x-auto'>
+          <table className='min-w-full divide-y divide-gray-700'>
+            <thead className='bg-gray-900'>
+              <tr>
+                {[
+                  'Trade',
+                  '🪙',
+                  'Question',
+                  '📅 Closes',
```

Comment:
What should happen on the API side if the market is already closed? (but not resolved yet as either YES/NO)?

Should the API return, "CLOSED - DATE" or still just the date?

E.g. should the front end be in charge of displaying, "CLOSED - DATE" and converting that if the date was in the past or should the back end serve that information?

Reply:
Resolved in other PR